data/json/items/comestibles/raw_fruit.json

```diff
@@ -472,279 +472,352 @@
+    "vitamins": [ [ "vitC", 2 ], [ "iron", 10 ] ]
   },
   {
     "type": "COMESTIBLE",
     "id": "peach",
     "name": { "str": "peach", "str_pl": "peaches" },
-    "weight": "163 g",
+    "weight": "150 g",
     "color": "yellow",
     "spoils_in": "3 days",
     "comestible_type": "FOOD",
     "symbol": "%",
     "quench": 4,
-    "calories": 63,
+    "calories": 58,
     "description": "This fruit's large pit is surrounded by its tasty flesh.",
-    "price": 140,
+    "price": 38,
     "price_postapoc": 100,
     "material": [ "fruit" ],
-    "volume": "250 ml",
+    "volume": "180 ml",
     "fun": 4,
     "flags": [ "SMOKABLE" ],
     "smoking_result": "dry_fruit",
-    "vitamins": [ [ "vitC", 12 ], [ "calcium", 1 ], [ "iron", 2 ] ]
+    "vitamins": [ [ "vitC", 11 ], [ "iron", 2 ] ]
   },
   {
     "type": "COMESTIBLE",
     "id": "watermelon",
     "name": { "str": "watermelon" },
-    "weight": "161 g",
+    "weight": "4518 g",
     "color": "green",
     "spoils_in": "7 days 12 hours",
     "comestible_type": "FOOD",
     "symbol": "%",
-    "quench": 28,
-    "//": "yes, it really does have that much water in it",
-    "calories": 48,
+    "quench": 826,
+    "//": "yes, it really does have that much water in it - 4131.7g in this size specifically",
+    "calories": 1355,
     "description": "A deep green melon, bigger than your head.  It is very juicy!",
-    "price": 400,
+    "price": 758,
     "price_postapoc": 600,
     "material": [ "fruit" ],
-    "volume": "5 L",
-    "charges": 20,
+    "volume": "7150 ml",
     "bashing": 2,
     "flags": [ "EATEN_COLD", "FREEZERBURN", "SMOKABLE" ],
+    "smoking_result": "dry_fruit",
+    "vitamins": [ [ "vitC", 406 ], [ "calcium", 24 ], [ "iron", 60 ] ]
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "watermelon_wedge",
+    "name": { "str": "watermelon wedge" },
+    "weight": "286 g",
+    "color": "green",
+    "spoils_in": "7 days 12 hours",
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "quench": 52,
+    "calories": 85,
+    "description": "A wedge of a watermelon.  Significantly more fitted to be consumed.",
+    "price": 48,
+    "price_postapoc": 40,
+    "material": [ "fruit" ],
+    "volume": "445 ml",
+    "flags": [ "EATEN_COLD", "FREEZERBURN", "SMOKABLE" ],
     "fun": 4,
     "smoking_result": "dry_fruit",
-    "vitamins": [ [ "vitC", 14 ], [ "calcium", 1 ], [ "iron", 2 ] ]
+    "vitamins": [ [ "vitC", 25 ], [ "calcium", 1 ], [ "iron", 4 ] ]
   },
   {
     "type": "COMESTIBLE",
     "id": "melon",
-    "name": { "str": "melon" },
-    "//": "based on a canteloupe",
-    "weight": "165 g",
+    "name": { "str": "cantaloupe melon" },
+    "weight": "552 g",
     "color": "light_green",
     "spoils_in": "7 days 12 hours",
     "comestible_type": "FOOD",
     "symbol": "%",
-    "quench": 4,
-    "calories": 54,
+    "quench": 20,
+    "calories": 187,
     "description": "A large and very sweet fruit.  Botanically speaking, all melons are actually berries.",
```

Comment:
A large and very sweet orange fruit.  Technically a berry, like all melons are.  You should find something to slice it up, as any other method of cracking open and eating it would be very messy.